src/bridge/cockpitdbusjson.c

```diff
@@ -46,6 +46,7 @@ typedef struct {
   GDBusConnection *connection;
   gboolean subscribed;
   guint subscribe_id;
+  gboolean allow_external;
```

Comment:
Wouldn't it be far less code to just make this a global? Since it really applies to all CockpitDBusJSON in a single process? That is, something like cockpit_bridge_local_address or similar variables?